provider/sourcegraph-search/index.test.ts

```diff
@@ -0,0 +1,48 @@
+import type { AnnotationsResult, CapabilitiesResult } from '@openctx/provider'
```

Comment:
this test will fail, just copy-pasta from hello world. Will update when we are happy with the APIs.